cltk/prosody/old_norse/verse.py

```diff
@@ -589,3 +591,57 @@ def find_alliteration(self):
         :return:
         """
         return Metre.find_alliteration(self)
+
+
+class PoetryTools:
+    def __init__(self, basic=True):
+        if basic:
+            self.syllabifier = Syllabifier(language="old_norse_ipa")
+            self.tr = Transcriber(DIPHTHONGS_IPA, DIPHTHONGS_IPA_class, IPA_class, old_norse_rules)
+            self.tagger = POSTag('old_norse')
+        else:
+            self.syllabifier = None
+            self.phonetic_transcriber = None
+            self.tagger = None
+
+
+class PoeticWord:
+    def __init__(self, text):
+        self.text = text
+        self.syl = []
+        self.length = []
+        self.stress = []
+        self.ipa_transcription = []
+
+    def apply_poetic_tool(self, poetry_tools: PoetryTools):
```

Comment:
We need a better name for this, like perhaps … `parse_word`?

Reply:
Yes, why not. I'll see if I find better.

Reply:
Done.